internal/app/wwctl/node/set/root.go

```diff
@@ -31,11 +31,11 @@ func GetCommand() *cobra.Command {
 	vars.nodeConf = node.NewConf()
 	baseCmd := &cobra.Command{
 		DisableFlagsInUseLine: true,
-		Use:                   "set [OPTIONS] PATTERN [PATTERN ...]",
+		Use:                   "set [OPTIONS] PATTERN",
 		Short:                 "Configure node properties",
 		Long:                  "This command sets configuration properties for nodes matching PATTERN.\n\nNote: use the string 'UNSET' to remove a configuration",
 		Aliases:               []string{"modify"},
-		Args:                  cobra.MinimumNArgs(0),
+		Args:                  cobra.ExactArgs(1), // require pattern as a mandatory arg
```

Comment:
@JasonYangShadow Would `MinimumNAgrs(1)` work, too? That way we could still supply multiple nodes as multiple "patterns."

Reply:
thanks! updated. `MinimumNArgs` should work better in this case.